rootfs/usr/local/bin/eks-update-kubeconfig

```diff
@@ -96,6 +113,9 @@ main() {
 		file_name $2 $role
 		return 0
 	fi
+	if [[ $1 =~ ^[[:alnum:]]+-[[:alnum:]]+-[[:alnum:]]+$ ]]; then
```

Comment:
Although this is probably OK, I prefer the more explicit statement of our convention:
```suggestion
	if [[ $1 =~ ^[^-]+-[^-]+-[^-]+$ ]]; then
```